src/main/java/com/github/tomakehurst/wiremock/client/ResponseDefinitionBuilder.java

```diff
@@ -262,6 +263,23 @@ private boolean isBinaryBody() {
   }
 
   private boolean isJsonBody() {
+    if (jsonBody != null) {
+        return true;
+    }
+    if(headers == null || stringBody == null) {
+        return false;
+    }
+    HttpHeader contentTypeHeader = headers.stream()
+        .filter(header -> header.keyEquals(ContentTypeHeader.KEY)).findFirst().orElse(null);
+    if(contentTypeHeader != null) {
+        boolean isJson = contentTypeHeader.hasValueMatching(new ContainsPattern("json"));
```

Comment:
@tomakehurst this may be one of those areas we'd talked about with respect to needing to do "proper" content-negotation, as I'd recommend we check that it is compatible with the media type subtype `*+json`